src/components/UserInfo/UserInfo.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import './UserInfo.scss';
+
+type Props = {
+  user: User,
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <div className="user-info">
+    <strong className="user-info__name">Name:</strong>
+    {' '}
+    <span className="user-info__name">{user.name}</span>
```

Comment:
better use css styles instead of `strong`

```suggestion
    <span className="user-info__name">
      {`Name: ${user.name}`}
     </span>
```